spk/radarr/src/dsm-control.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+# Package
+PACKAGE="radarr"
+DNAME="Radarr"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="${PACKAGE}"
+USER_HOME="$(eval echo ~$USER)"
```

Comment:
Please remove (so I don't have to do this afterwards ;)). Where `USER_HOME` is used, please use `${INSTALL_DIR}/var` instead